cmd/admin-heal.go

```diff
@@ -99,10 +101,12 @@ func (u healObjectMessage) JSON() string {
 // checkAdminHealSyntax - validate all the passed arguments
 func checkAdminHealSyntax(ctx *cli.Context) {
 	if len(ctx.Args()) != 1 {
-		cli.ShowCommandHelpAndExit(ctx, "heal", 1) // last argument is exit code
+		cli.ShowSubcommandHelp(ctx)
```

Comment:
I found a hacky change but it seems that my approach doesn't work well with cli package.


```
  1 package main
  2 
  3 import (
  4         "fmt"
  5         "os"
  6 
  7         "github.com/urfave/cli"
  8 )
  9 
 10 func main() {
 11 
 12         adminServiceCmd := cli.Command{
 13                 Name: "service",
 14                 Action: func(c *cli.Context) {
 15                         fmt.Printf("Run '%s'\n", c.Command.Name)
 16                 },
 17         }
 18 
 19         adminCmd := cli.Command{
 20                 Name:        "admin",
 21                 Subcommands: []cli.Command{adminServiceCmd},
 22                 Action: func(c *cli.Context) {
 23                         fmt.Printf("Run '%s'\n", c.Command.Name)
 24                 },
 25         }
 26 
 27         app := cli.NewApp()
 28         app.Name = "lookup"
 29         app.Commands = []cli.Command{adminCmd}
 30 
 31         app.Run(os.Args)
 32 }
```
```
$ ./a.out admin
Run ''
$ ./a.out admin service
Run 'service'
```

If you type `$ ./a.out admin`, it doesn't show the command name as I expect.